hadoop/src/main/java/org/apache/carbondata/hadoop/util/CarbonInputFormatUtil.java

```diff
@@ -48,7 +48,7 @@
 
   public static <V> CarbonFileInputFormat<V> createCarbonFileInputFormat(
       AbsoluteTableIdentifier identifier, Job job) throws IOException {
-    CarbonFileInputFormat<V> carbonInputFormat = new CarbonFileInputFormat<V>();
+    CarbonFileInputFormat<V> carbonInputFormat = new CarbonFileInputFormat<>();
```

Comment:
Looks like this method is Unused and LOGGER also

Reply:
remove Unused variable and method